### PR TITLE
Feature/ui session with ingo

### PR DESCRIPTION
**Pull request by 99999989**
* add icon in header for mobile view
* align main navigation entries with the query-search-panels
* open link for arbeit.swiss in separate tab (mobile-menu)
* fix top alignments on mobile view (after effect of changing header height)
* company box on candidate detail and job-ad detail (EDGE and firefox): email address is not broken (break-word: all  not working)
* manage-job-ads: align filter-button with title, remove margin below the title (same margin as “Stelle melden”)
* header (lang change + user icon): remove dropdown indicator on mobile view, keep it for desktop
* color-grey-black instead of black (everywhere) 
* ellipsis icon (actions)
  1. job-ad-management-table row 61: fix aria-label text to “Actions”
  2. everywhere where icon “ellipsis-v” is used, add tooltip “Actions” (and aria-label)

### CODE REVIEW

File: alv-portal-ui/src/app/shared/layout/header/header.component.html [hunk 1]
```diff
@@ -3,18 +3,28 @@
   <h1 class="sr-only">{{'portal.header.header-label' | translate}}</h1>
 
   <div class="header p-0 d-flex align-items-center">
+    <div class="mobile-logo d-block d-lg-none text-center position-absolute w-100">
+      <img class="mr-1"
+           alt="Logo arbeit.swiss"
```
Comment (ganqqwerty) [inline alv-portal-ui/src/app/shared/layout/header/header.component.html]: i18n

File: alv-portal-ui/src/app/shared/layout/layout-constants.enum.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export enum LayoutConstants {
    *              3rem        + 3rem           + 46px                = 142px
    *              48px        + 48px           + 46px                = 142px
    */
-  STICKY_TOP_SEARCH = '142px',
+  STICKY_TOP_SEARCH = '118px',
```
Comment (ganqqwerty) [inline alv-portal-ui/src/app/shared/layout/layout-constants.enum.ts]: can you start making them into variables?

File: alv-portal-ui/src/app/widgets/manage-job-ads-widget/job-ad-management-table/job-ad-management-table.component.scss [hunk 3]
```diff
@@ -3,67 +3,71 @@
...
 
-td {
-  em {
-    padding: 0 0.125rem;
-    background-color: $color-light-yellow;
+    &:hover {
+      border-left: 5px solid $color-primary;
+      color: $color-primary;
+      cursor: pointer;
+      //box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.35);
```
Comment (ganqqwerty) [inline alv-portal-ui/src/app/widgets/manage-job-ads-widget/job-ad-management-table/job-ad-management-table.component.scss]: xx

File: alv-portal-ui/src/app/shared/layout/header/header.component.html (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/shared/layout/header/header.component.html]: done

File: alv-portal-ui/src/app/shared/layout/layout-constants.enum.ts (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/shared/layout/layout-constants.enum.ts]: It's already a variable, but in .ts  and not .scss

File: alv-portal-ui/src/app/widgets/manage-job-ads-widget/job-ad-management-table/job-ad-management-table.component.scss (hunk 3 above)
Comment (99999989) [reply alv-portal-ui/src/app/widgets/manage-job-ads-widget/job-ad-management-table/job-ad-management-table.component.scss]: removed

**Pull request merged**